telink: crypto: mbedtls: Update MbedTLS testing process for TL721X.

pulin1103 opened the pull request:
- add pke_x25519_point_mul dummy func to compile.
- rename eccp_curve_t structure.
- update TL721X pke header file.

liucoldstar commented on tlsr9/drivers/TL721X/lib/include/pke/pke.h:
```diff
@@ -29,9 +29,22 @@ extern "C" {
-#define SUPPORT_C25519
-
-
+// #define SUPPORT_C25519
```
这个宏不需要启用吗？

pulin1103 replied:
TL721X不支持x25519，没有x25519相关函数接口，此处不启用；TL321X支持x25519，所以启用。

liucoldstar commented on tlsr9/crypto/mbedtls/ecp_b9x_backend.c:
```diff
@@ -216,18 +216,6 @@ static eccp_curve_t secp192k1_curve_dat = {
 #define MONT_P_H            p_h
 #define MONT_P_N1           p_n1
 #define MONT_A24            a24
-#elif CONFIG_SOC_RISCV_TELINK_B95
```
你提Zephyr的PR的时候看看，这部分代码是不是还不能移除？

pulin1103 committed:
telink: crypto: mbedtls: Update MbedTLS testing process for TL721X.

- add pke_x25519_point_mul dummy func to compile.
- rename eccp_curve_t structure.
- update TL721X pke header file.

Signed-off-by: pulin1103 <zhenghuan.zhang1103@gmail.com>

liucoldstar commented on tlsr9/crypto/mbedtls/ecp_tlx_backend.c:
```diff
@@ -34,24 +34,42 @@ LOG_MODULE_REGISTER(ecc_tlx, CONFIG_MBEDTLS_LOG_LEVEL);
  * HW unit curve data constants definition
  ****************************************************************/
 
+#if CONFIG_SOC_RISCV_TELINK_TL321X
```
下面这几个宏除了ECCP_P_N，应该可以直接共用

pulin1103 replied:
是可以复用，这样写是为了代码看起来更整齐，以后新的芯片只需要再添加一套定义就行了
